src/jsonrpc-discover.json

```diff
@@ -34,6 +34,27 @@
       }
     },
 
+    {
+      "name": "rebind",
+      "summary": "Rebinds the server",
```

Comment:
"Changes the address server is listening to."

Most people will have no idea what bind even is. :)